runtime-components/non-repudiation-perf/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_benchmark_jmh",
+)
+load(
+    "@scala_version//:index.bzl",
+    "scala_major_version_suffix",
+)
+
+da_scala_benchmark_jmh(
```

Comment:
This rule unfortunately does not support `scala_deps` and `runtime_deps` (something we may want to work on at a certain point).

Reply:
How’s that https://github.com/digital-asset/daml/pull/8928

Reply:
merged the PR, i’ll leave it up to you if you want to rebase on top of it or do the cleanup separately.

Reply:
Well, this was easier than what I thought. 😄 Thanks.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/8916/commits/2ab5c17ba236690830f2b60e2092149f41520e88